### PR TITLE
Add search

### PR DESCRIPTION
**Pull request by imaustink**
Reverts canjs/canjs#3399

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "github": "https://github.com/canjs/canjs"
   },
   "dependencies": {
+    "bit-docs-glob-finder": "0.0.6",
```
Comment (chasenlehara) [inline package.json]: Out of curiosity, why was this added?

File: package.json [hunk 2]
```diff
@@ -186,6 +189,6 @@
       "2.3.31": "https://v2.canjs.com"
     },
     "parent": "canjs",
-    "minifyBuild": true
+    "minifyBuild": false
```
Comment (chasenlehara) [inline package.json]: This will need to be changed back.

File: package.json (hunk 1 above)
Comment (imaustink) [inline package.json]: Strange... I installed that when I was trying to debug using this with the local copies of the plugins, but I didn't use the save flag so I'm not sure how this got added.

Comment (imaustink) [inline package.json]: Actually, this is needed. Otherwise and error is thrown: 
```
/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:155
                throw e;
                ^

TypeError: siteConfig.finder is not a function
    at generateOne (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/generate/generate.js:49:36)
    at generateAndWatch (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/generate/generate.js:82:10)
    at /Users/austin/bitovi/canjs/node_modules/bit-docs/main.js:6:10
    at _fulfilled (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:854:54)
    at self.promiseDispatch.done (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:883:30)
    at Promise.promise.promiseDispatch (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:816:13)
    at /Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:624:44
    at runSingle (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:137:13)
    at flush (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
```

Comment (chasenlehara) [inline package.json]: I would like to better understand what changed that required this to be included.

[FYI, as of npm 5, any install will `--save` by default.]

Comment (chasenlehara) [inline package.json]: I could not reproduce the `siteConfig.finder is not a function` issue so I removed it.
